package.json

```diff
@@ -82,6 +82,7 @@
 		"feedme": "^2.0.2",
 		"helmet": "^6.0.0",
 		"iconv-lite": "^0.6.3",
+		"isomorphic-fetch": "^3.0.0",
```

Comment:
Isn't that a dev dependency?

Reply:
Related to the [problem explained below](https://github.com/MichMich/MagicMirror/pull/2957#discussion_r1004858313) 😊 This won't be in the final version of the PR.